zson/analyzer.go

```diff
@@ -503,7 +503,7 @@ func (a Analyzer) convertEnum(zctx *zed.Context, val *astzed.Enum, cast zed.Type
 			}, nil
 		}
 	}
-	return nil, fmt.Errorf("symbol %q not a member of type %q", val.Name, FormatType(enum))
+	return nil, fmt.Errorf("symbol %q not a member of type %q", val.Name, String(enum))
```

Comment:
String's OK but I vote for using FormatType everywhere instead since it's type-safe.